internal/datastore/mysql/datastore.go

```diff
@@ -527,7 +527,14 @@ func (mds *Datastore) seedDatabase(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	defer common.LogOnError(ctx, tx.Rollback)
+
+	var committed bool
```

Comment:
maybe we should have a function that takes another function as a closure to perform work, but also do the rollback on failure... thoughts?